html-forms/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <div>
+    <div>
+      <label for="email">Email</label>
+    </div>
+    <div>
+      <input type="text" id="email" name="email">
+    </div>
+  </div>
+  <div>
+    <div>
+      <label for="codeCool">Why is coding cool?</label>
+    </div>
+    <div>
+      <select id="codeCool" name="codeCool">
+        <option value="select">Select an option</option>
```

Comment:
For the starting "default" option we usually leave the `value` attribute as an empty string